.buildkite/pipeline.yml

```diff
@@ -0,0 +1,86 @@
+---
+# Constants
+RUBY_VERSION: &RUBY_VERSION 2.6.1
+UNIT_TEST_CONFIG: &UNIT_TEST_CONFIG
+  timeout_in_minutes: 5
+  agents:
+    queue: ruby-mysql
+    ruby: *RUBY_VERSION
+    mysql: 5.7
+db_setup_commands: &db_setup_commands
+  - bundle exec rake db:setup
+bundle_install_commands: &bundle_install_commands
+  - bundle install --without development
+  - bundle exec appraisal install --without development
```

Comment:
To help save time, we can skip using appraisal to run the commands and set the `BUNDLE_GEMFILE` environment variable instead. This saves a lot of time on the builds making it so that each step only has to install a single bundle environment rather than all of them.